srcpkgs/scenefx/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'scenefx'
+pkgname=scenefx
+version=0.1
+revision=1
+build_style=meson
+configure_args="-Dwerror=false -Db_ndebug=false"
+hostmakedepends="pkg-config wayland-devel scdoc"
+makedepends="wlroots0.17-devel"
+depends="libdrm pixman"
+short_desc="Drop-in replacement for the wlroots scene API"
+maintainer="Christopher K. 'Shmish' Schmitt <me@shmish.dev>"
+license="MIT"
+homepage="https://github.com/wlrfx/scenefx"
+changelog="https://github.com/wlrfx/scenefx/releases/"
+distfiles="https://github.com/wlrfx/scenefx/archive/refs/tags/${version}.tar.gz"
+checksum=f5c889f4c826a4216628bf1e7e48071fc33e7774b5e3d51e6fee6e571e420827
+
+post_install() {
+	vlicense LICENSE
+}
```

Comment:
needs a -devel subpackage

Reply:
added, and `swayfx` package modified to use `-devel` dependency